user-onboarding/src/main/java/br/com/edsilfer/android/user_onboarding/presenter/FragmentPlaceHolder.kt

```diff
@@ -30,9 +30,11 @@ class FragmentPlaceHolder : Fragment() {
     }
 
     override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-        val rootView = inflater!!.inflate(R.layout.fragment_pager_item, container, false)
         val page = arguments.getSerializable(ARG_PAGE) as Page
 
+        val layoutResource = page.contentLayoutResource
```

Comment:
Can we directly assign ```page.contentLayoutResource``` on inflate method? No need for ```layoutResource ```  var. Just something that would be caught on Sonnar.